docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-dynamic-user-task-API-proc.adoc

```diff
@@ -0,0 +1,60 @@
+[id='case-management-dynamic-user-task-API-proc']
+= Creating a dynamic user task using the REST API
+
+You can create a dynamic user task during case run time using the REST API. To create a dynamic user task, the following information must be given:
+
+* task name
+* task description (optional, but recommended)
+* actors
+* groups
+* input data
+
+Use the following procedure to create a dynamic user task for the IT Orders sample project available in {CENTRAL} using the Swagger REST API tool. The same endpoint can be used for REST API without Swagger.
```

Comment:
Consider reordering: 
 +Use the following procedure to use the Swagger REST API to create a dynamic user task  for the IT Orders sample project available in {CENTRAL} .